docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -18,23 +18,25 @@ To make your validator node and validator fullnode deployment hassle-free, make
 ## Hardware requirements
 
 For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
+For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
+TPS on mainnet. In order to evaluate that, you can use the command (https://gist.github.com/igor-aptos/298c87c0a5f095bdae4716b526d82520)
+It has two benchmarks, a shorter one (~10m) and a large db one (few hours). Once the script finishes, it will print out a table like: https://gist.github.com/igor-aptos/97d43db4f460ab4b29e4902cb3d31954, and column "t/s" is your achieved "transactions per second". Evaluation criteria is encoded in the script, and will warn you after the table if you've not met any of the criteria.
 
 - **CPU**:
```

Comment:
I'd update this to "CPU requirement reference", and update 60 cores to 32 instead, it seems in our spreadsheet people does reach similar result with 32 core machines too. so they can start testing with lower spec and upgrade accordingly.